app/views/candidate_interface/references/review/confirm_destroy_reference.html.erb

```diff
@@ -13,6 +13,12 @@
         <%= t('page_titles.references_delete_reference') %>
```

Comment:
Thinking we’ll probably want a more explicit title in this case: ‘Are you sure you want to delete this selected reference’.